src/main/java/creativei/enums/ExceptionType.java

```diff
@@ -8,7 +8,8 @@
     DUPLICATE_VALUE(1002, "Value already exists"),
     DATABASE_EXCEPTION(1003, "Exception occured while adding values to database"),
     INVALID_METHOD_PARAM(1004,"Invalid Method Request "),
-    DATA_NOT_AVAILABLE(1005,"No Data Is Availabe");
+    NO_RECORD_FOUND(1005,"No Record Found"),
+    DATA_NOT_AVAILABLE(400,"Invalid Inquiry Status");
```

Comment:
Why do we need 2 different Exceptions in this case?

Reply:
record not found  for if id  is not there and invalid inquiry status is for if required status is not found.
Sir can i do something like " no data found for requested parameter".THis can be common for both scenario.

Reply:
Yes this is fine

Reply:
issue solved in commit d5326cd